lib/clean.js

```diff
@@ -1,15 +1,17 @@
 'use strict'
 
-const rm = require('rimraf')
-const log = require('npmlog')
+const fs = require('fs')
```

Comment:
Iirc rimraf is used over fs because it is better cross arch?

Reply:
fwiw, `npm` has moved from `rimraf` to `fs/promises` and not seen issues across architectures/platforms. happy to keep `rimraf` in here though if that makes sense

Reply:
by all means no, less deps the better :)